### PR TITLE
Iteration2/hydration tdd

### PR DESCRIPTION
**Pull request by demaceo**
#### What does  this PR do?
Refactors the `HydrationRepository` to include methods, such as:
- `calculateAverageWeeklyOunces()`,` calculateAverageDailyOunces()`, `calculateAverageTotalOunces()`, & `sumDailyOunces()`
#### Where should the reviewer start?
View **hydrationRepository.js** & **HydrationRepository-test.js**
#### How should this be manually tested?
N/A
#### Any background context you want to provide?
Currently the tests are unable to run on my local machine.
Additionally, the `calculateAverageWeeklyOunces()` is not finished.

### CODE REVIEW

File: src/hydrationRepository.js [hunk 1]
```diff
@@ -1,27 +1,63 @@
...
-    this.ouncesRecord.unshift({
-      [date]: amount
-    });
-    if (this.ouncesRecord.length) {
-      this.ouncesAverage = Math.round((amount + (this.ouncesAverage * (this.ouncesRecord.length - 1))) / this.ouncesRecord.length);
-    } else {
-      this.ouncesAverage = amount;
-    }
+
+  // addDailyOunces(userID, date, numOunces) {
```
Comment (peytonjo) [inline src/hydrationRepository.js:6]: Looks good! Glad we are getting the hydration repo set up

File: test/HydrationRepository-test.js [hunk 2]
```diff
@@ -5,14 +5,10 @@ import HydrationRepository from '../src/HydrationRepository';
 import User from '../src/User';
 
 describe.only('HydrationRepository', function() {
-  let todaysDate;
+  // let todaysDate;
```
Comment (peytonjo) [inline test/HydrationRepository-test.js:8]: these tests look good! we should probably create some more

**Review by JWoltjen — APPROVED** [review]
Nice work getting everything started on the HydrationRepository class Demaceo!

File: src/activityRepository.js [hunk 3]
```diff
@@ -1,6 +1,11 @@
-class ActivityRepository {
+class ActivtyRepository {
   constructor() {
     this.record = [];
```
Comment (peytonjo) [inline src/activityRepository.js:3]: Most of this stuff looks commented out we may need to just delete it if its dead code

**Review by peytonjo — APPROVED** [review]

**Pull request merged**